CoEpi/ui/healthquiz/HealthQuizViewModel.swift

```diff
@@ -5,29 +5,31 @@ class HealthQuizViewModel {
     weak var delegate: HealthQuizViewModelDelegate?
 
     private let symptomRepo: SymptomRepo
+    private let questionsRelay: BehaviorRelay<[Question]>
 
-    private var questions: [Question]
-    
     private(set) var rxQuestions: Driver<[Question]>
     
     init(container: DependencyContainer) {
         self.symptomRepo = try! container.resolve()
-        questions = symptomRepo.symptoms().map{ Question(symptom: $0) }
+        questionsRelay = BehaviorRelay(value: symptomRepo.symptoms()
```

Comment:
Since you're already working on this, you could move the `Relay` here as well to the repo. I didn't do it because we didn't need to edit the symptoms yet, but we will need it.

Reply:
I'm not sure we want to do that. The symptom repo is the canonical source of all Symptoms, vs the behavior relay here is for Questions. For example, if a user selected a few answers in a quiz, and then did a quiz again, we wouldn't want the first quiz to impact the 2nd quiz (e.g. missing questions/symptoms, prechecked ones, etc.)

Questions drive view state, vs symptoms are the full set of possible symptoms, afaik.

Reply:
I didn't express myself well, I mean basically only to make the symptoms reactive. But I realize now that it's not trivial to wire this "rx-way" with the question inputs. You can let it for now like this. We will have to wire it in any case later, if/when we get the symptoms from a backend.